bin/sentry-wrap.py

```diff
@@ -69,30 +74,37 @@ def wrap_process(ctx, timeout, cmd):
 
     cmd = " ".join(cmd)
     cmd_args = shlex.split(cmd)
-    click.echo(f"Running: {cmd_args}")
+    if verbose:
+        click.echo(f"Running: {cmd_args}")
 
     try:
-        ret = subprocess.run(cmd_args, capture_output=True, timeout=timeout)
+        ret = subprocess.run(
+            cmd_args,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
```

Comment:
I switched this from `capture_output=True` which captures stdout and stderr separately to this which captures them both in the stdout pipe, but multiplexed so the lines are in the same order that you'd see them in a terminal. That's a lot easier to understand when debugging things.

Reply:
I can confirm I'm now seeing stderr and stdout in the `make run` output now locally. I can also now see the disk manager logs:

```
tecken-web-1         | [2023-11-29 21:40:37 +0000] [29] [ERROR] Worker (pid:42) exited with code 1
tecken-web-1         | [2023-11-29 21:40:37 +0000] [29] [ERROR] Worker (pid:42) exited with code 1.
tecken-web-1         | [2023-11-29 21:40:37 +0000] [24812] [INFO] Booting worker with pid: 24812
tecken-web-1         | WARNING 2023-11-29 21:40:41,662 - webapp - tecken Auth0 blocked check disabled
tecken-web-1         | INFO 2023-11-29 21:41:17,783 - disk_manager - tecken.remove_orphaned_files expires: 15 (minutes)
tecken-web-1         | INFO 2023-11-29 21:41:17,788 - disk_manager - tecken.remove_orphaned_files watchdir: '/tmp/uploads'
```